pkg/minikube/cruntime/docker.go

```diff
@@ -127,7 +143,12 @@ func (r *Docker) KillContainers(ids []string) error {
 		return nil
 	}
 	glog.Infof("Killing containers: %s", ids)
-	return r.Runner.Run(fmt.Sprintf("docker rm -f %s", strings.Join(ids, " ")))
+	args := append([]string{"rm", "-f"}, ids...)
```

Comment:
ids should remain separate arguments.

Reply:
@tstromberg I don't undrestand what you mean. do you mind explaining ?